alacritty/src/renderer/mod.rs

```diff
@@ -22,23 +23,29 @@ use std::sync::mpsc;
 use std::time::Duration;
 
 use fnv::FnvHasher;
+
 use font::{
     self, BitmapBuffer, FontDesc, FontKey, GlyphKey, Rasterize, RasterizedGlyph, Rasterizer,
 };
+
 use log::{error, info};
+
```

Comment:
Not really a fan of these newlines. Dunno why, they just irk me (not a real reason to change anything, I know).

Going off some of the other imports I looked at, it might look better if the `log` and `fnv` imports were grouped together and `font` separated by a newline, instead of all 3 separated by a newline.